lib/plugins/orionUrls.js

```diff
@@ -39,6 +39,8 @@ module.exports = [
   ['POST', /\/(v1|ngsi10)\/unsubscribecontext$/, 'subscribe'],
   ['POST', /\/(v1\/registry|ngsi9)\/registercontext$/, 'register'],
   ['POST', /\/(v1\/registry|ngsi9)\/discovercontextavailability$/, 'discover'],
+  ['POST', /\/(ngsi10|v1)\/contexttypes$/, 'read'],
```

Comment:
In addition, the action should be added to the table in the documentation, isn't it?